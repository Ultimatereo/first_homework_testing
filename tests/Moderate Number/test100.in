5
-368522117299 176276958353 -758683472309 799661933628 -825907418927 -672817830757
-718931347474 -456172332642 409284923683 339551272850 -815395857337 28611491355
663393135823 -540695630479 -995567440949 -139866501388 -584937378505 -516941239346
-565853737871 -175955843203 -391211572359 699284124059 -405521215463 -984226531233
673938236266 -439991587207 -549356087739 208581827277 520496457400 -334571423391
