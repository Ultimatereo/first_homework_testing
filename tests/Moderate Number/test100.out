1.67141
0.608266
1.66999
0.588193
-0.765881
